snap/snapcraft.yaml

```diff
@@ -25,25 +23,17 @@ plugs:
 
 package-repositories:
   - type: apt
-    ppa: data-platform/patroni
-  - type: apt
-    ppa: data-platform/pgbackrest
-  - type: apt
-    ppa: data-platform/pgbouncer
-  - type: apt
-    ppa: data-platform/postgres-exporter
-  - type: apt
-    ppa: data-platform/pgbouncer-exporter
-  - type: apt
-    ppa: data-platform/pgvector
-  - type: apt
-    ppa: data-platform/python-pysyncobj
+    ppa: data-platform/postgresql-16-timescaledb-tsl
   - type: apt
     ppa: data-platform/pgbackrest-exporter
   - type: apt
-    ppa: data-platform/postgresql-14-timescaledb
+    ppa: data-platform/pgbackrest
```

Comment:
Would be good to note why do we need pgbackrest from PPA here.

Reply:
PPA is more recent: [2.53](https://launchpad.net/~data-platform/+archive/ubuntu/pgbackrest) vs. [2.50](https://packages.ubuntu.com/noble/pgbackrest).